src/main.ts

```diff
@@ -62,5 +62,9 @@ export { utils };
  */
 export type { Feature, Point, PointType, LineString };
 
-export { LoadingControl };
-export type { LoadingControlConfiguration };
+/**
+ * @deprecated in favor of {@link LoadingIndicatorControl}
+ */
+export { LoadingIndicatorControl as LoadingControl };
+export { LoadingIndicatorControl };
+export type { LoadingIndicatorControlConfiguration };
```

Comment:
Just do a breaking change. I think you are the only user at the moment... Or is the plan to do the breaking change when we go to version 1?

Reply:
Yes, I think 1.0 is a right place to do so.
> Just do a breaking change.
You never know :) Also, there's at least one project that shows up in the "used by" section at the front page.

Reply:
@wipfli could you please merge and release a new patch version as you have time and as you think everything's resolved? Or if there are other questions arise, then please ask, I'll be happy to answer.

Reply:
Sure. https://github.com/maplibre/maplibre-gl-directions/runs/6278240404?check_suite_focus=true